egui_glium/src/http.rs

```diff
@@ -3,9 +3,17 @@ pub use epi::http::{Request, Response};
 /// NOTE: Ok(..) is returned on network error.
 /// Err is only for failure to use the fetch api.
 pub fn fetch_blocking(request: &Request) -> Result<Response, String> {
-    let Request { method, url } = request;
+    let Request { method, url, body } = request;
 
-    let resp = ureq::request(method, url).set("Accept", "*/*").call();
+    let resp = match method.as_str() {
+        "GET" => ureq::get(url).set("Accept", "*/*").call(),
+        "POST" => ureq::post(url)
+            .set("Accept", "*/*")
+            .set("Content-Type", "text/plain; charset=utf-8")
+            .send_string(body),
+        _ => return Err("method not implemented".to_string()),
+        // TODO: PUT only sends in binary form.
+    };
```

Comment:
```suggestion
    let mut req = ureq::request(method, url).set("Accept", "*/*");
    if !body.is_empty() {
        req.set("Content-Type", "text/plain; charset=utf-8");
        req.send_string(body);
    }
    let resp = req.call();
```

this way we will support the other methods too!